Views/Solutions/Show.cshtml

```diff
@@ -1,31 +1,41 @@
 ﻿@model Developer_Toolbox.Models.Solution
-<section class="p-3">
-    <h1 class="">Solution:</h1>
-    <br />
 
-    <div class="card w-50">
```

Comment:
Displaying the solutions in cards takes a lot of spaces and becomes difficult to understand for the user. You should probably list them, including only some relevant information.

Reply:
I'll try now to fix this